pycon/program_export.py

```diff
@@ -107,6 +107,7 @@ def write(self, filename, objects, fields=None):
             csvwriter.writerow(dict([(fieldname, display(fieldname))
                                      for fieldname in fieldnames]))
             for obj in objects:
+                print("Writing a row to %s" % filename)
```

Comment:
Is there a reason for these to be print statements rather than logging statements?